app/src/main/java/com/spoony/spoony/presentation/register/componet/TopLinearProgressbar.kt

```diff
@@ -29,31 +29,38 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
 
+private const val DEFAULT_ANIMATION_DURATION = 300
+
 @Composable
 fun TopLinearProgressBar(
-    currentStep: Int
+    currentStep: Float,
+    totalSteps: Float = 3f,
+    animationDuration: Int = DEFAULT_ANIMATION_DURATION,
+    progressColor: Color = SpoonyAndroidTheme.colors.main400,
+    trackColor: Color = SpoonyAndroidTheme.colors.gray100,
+    modifier: Modifier = Modifier
 ) {
     val progress = remember { Animatable(0f) }
```

Comment:
p2) modifier 선언 순서 지켜주세요 :)
p3) progress InitialStep도 디폴트 0f로 넣어주는건 어떨까요?
```suggestion
    currentStep: Float,
    modifier: Modifier = Modifier
    initialSteps: Float = 0f,
    totalSteps: Float = 3f,
    animationDuration: Int = DEFAULT_ANIMATION_DURATION,
    progressColor: Color = SpoonyAndroidTheme.colors.main400,
    trackColor: Color = SpoonyAndroidTheme.colors.gray100,

) {
    val progress = remember { Animatable(initialSteps) }
```